kbase-extension/static/kbase/js/common/lang.js

```diff
@@ -23,8 +23,53 @@ define(['bluebird'], (Promise) => {
         });
     }
 
+    /**
```

Comment:
Both of the new functions in this file were found copied and pasted in at least three different other modules. I just consolidated them here as this seems to be a semi-general utility module.

Reply:
😖